cpp/doxygen/main_page.md

```diff
@@ -137,6 +137,13 @@ Note that if you're reading a large file that has been split into multiple reads
 
 These settings can also be controlled by `defaults::http_max_attempts()`, `defaults::http_max_attempts_reset()`, `defaults::http_status_codes()`, and `defaults::http_status_codes_reset()`.
 
+#### Remote Verbose (KVIKIO_REMOTE_VERBOSE)
+For debugging HTTP requests, you can enable verbose output that shows detailed information about HTTP communication including headers, request/response bodies, connection details, and SSL handshake information.
+
+Set the environment variable `KVIKIO_REMOTE_VERBOSE` to `true`, `on`, or `yes` (case-insensitive) to enable verbose output. Otherwise, verbose output is disabled by default.
```

Comment:
How about `1`? That is my favorite way to enable things.

Reply:
Yep, based on https://github.com/rapidsai/kvikio/blob/f4e022e9fcb768421c6e6ab5dddb0611b746fac8/cpp/src/defaults.cpp#L41-L42.

Updated the docs to note that.